roles/openshift_node/defaults/main.yml

```diff
@@ -168,37 +166,10 @@ journald_vars_to_replace:
 - { var: ForwardToSyslog, val: no }
 - { var: ForwardToWall, val: no }
 
-# NOTE
-# r_openshift_node_*_default may be defined external to this role.
-# openshift_use_*, if defined, may affect other roles or play behavior.
-openshift_node_use_openshift_sdn_default: "{{ openshift_use_openshift_sdn | default(True) }}"
-openshift_node_use_openshift_sdn: "{{ openshift_node_use_openshift_sdn_default }}"
-
-openshift_node_sdn_network_plugin_name_default: "{{ os_sdn_network_plugin_name }}"
-openshift_node_sdn_network_plugin_name: "{{ openshift_node_sdn_network_plugin_name_default }}"
-
-openshift_node_use_calico_default: "{{ openshift_use_calico | default(False) }}"
-openshift_node_use_calico: "{{ openshift_node_use_calico_default }}"
-
-openshift_node_use_nuage_default: "{{ openshift_use_nuage | default(False) }}"
-openshift_node_use_nuage: "{{ openshift_node_use_nuage_default }}"
-
-openshift_node_use_contiv_default: "{{ openshift_use_contiv | default(False) }}"
-openshift_node_use_contiv: "{{ openshift_node_use_contiv_default }}"
-
-openshift_node_use_kuryr_default: "{{ openshift_use_kuryr | default(False) }}"
-openshift_node_use_kuryr: "{{ openshift_node_use_kuryr_default }}"
-
 openshift_node_data_dir_default: "{{ openshift_data_dir | default('/var/lib/origin') }}"
 openshift_node_data_dir: "{{ openshift_node_data_dir_default }}"
 
 openshift_node_config_dir_default: "/etc/origin/node"
 openshift_node_config_dir: "{{ openshift_node_config_dir_default }}"
 
-openshift_node_image_config_latest_default: "{{ openshift_image_config_latest | default(False) }}"
-openshift_node_image_config_latest: "{{ openshift_node_image_config_latest_default }}"
-
-
 openshift_node_use_instance_profiles: False
-
-openshift_node_use_persistentlocalvolumes: "{{ openshift_persistentlocalstorage_enabled | default(False) | bool }}"
```

Comment:
openshift_node_use_persistentlocalvolumes looks like it never worked in a shipped release.  Only in 3.10 and master, and went into node.yaml.j2 (old node config).

We need to get docs cleaned up.  Relevant commit ID: 8eedacf0d1f2103d52250cc30bbe890cf71370a8

Reply:
@vrutkovs yeah, probably doc bug.  This was added in 3.10, then we refactored some stuff and apparently immediately broke it (or just didn't account for bootstrapping whatsoever).